docs/src/docs/asciidoc/getting-started.adoc

```diff
@@ -163,25 +167,29 @@ the configuration are described in the following listings:
 	}
 
 	asciidoctor { <6>
-		inputs.dir snippetsDir <7>
```

Comment:
Why has this been removed?

Reply:
Because according to the [Gradle documentation](https://docs.gradle.org/current/javadoc/org/gradle/api/tasks/TaskInputs.html#dir-java.lang.Object-), All files found under this snippetDir are treated as input files for this task. Is it what you want?

Reply:
Yes. The snippets need to be treated as inputs so that the `asciidoctor` task will run if the snippets change.

Reply:
So you want I put it back, correct?

Reply:
Yes please.

Reply:
Done!